src/components/select/select-list/select-list.spec.tsx

```diff
@@ -1067,6 +1068,13 @@ describe("SelectList", () => {
       expect(optionChildren.length).toBeGreaterThanOrEqual(5);
       expect(optionChildren.length).toBeLessThan(10);
     });
+
+    it("when an option is selected, it is always in the DOM even when out of view", () => {
```

Comment:
question (minor): is it worth having a check on visibility here as well?

Reply:
is that possible in a Jest/Enzyme test? If this were a cypress/Playwright test I'd agree that asserting that it isn't visible would be useful - but it isn't (and I'm testing this differently in Cypress, by just running the accessibility checks).

Reply:
No problem it's "nice" rather then "need" to have imo